src/validator.rs

```diff
@@ -979,11 +969,11 @@ mod tests {
 
         let report = validator.validate().unwrap();
 
-        assert!(report.errors.is_none());
+        assert!(report.is_none());
     }
 
     #[test]
-    fn test_literal_text_mismatch() {
+    fn test_literal_paragraph_mismatch() {
```

Comment:
These tests all look the same; may be worth the price of a small generalization?